v4l2-request-test.c

```diff
@@ -306,9 +330,19 @@ int main(int argc, char *argv[])
 		case '?':
 			print_help();
 			goto error;
+		default:
+			printf("?? getopt returned character code 0%o ??\n", opt);
```

Comment:
Please make the error message consistent with the rest of the code base.

Reply:
WIP